airflow/jobs/scheduler_job.py

```diff
@@ -1120,11 +1125,9 @@ def _verify_integrity_if_dag_changed(self, dag_run: DagRun, session=None):
         dag_run.verify_integrity(session=session)
 
     def _send_dag_callbacks_to_processor(self, dag: DAG, callback: Optional[DagCallbackRequest] = None):
-        if not self.processor_agent:
-            raise ValueError("Processor agent is not started.")
         self._send_sla_callbacks_to_processor(dag)
         if callback:
-            self.processor_agent.send_callback_to_execute(callback)
+            self.executor.send_callback_to_execute(callback)
```

Comment:
The `to_execute` suffix here doesn't make as much sense now it's on the executor -- it's implied.

So I would say we should change this to one of/something like

```suggestion
            self.executor.queue_callback(callback)
            self.executor.send_callback(callback)
            self.executor.execute_callback(callback)
```

WDYT?

Reply:
I agree. Let' use "send_callback" as we use "send" verb in other places.